sources/packages/workflow/BPMN/pt-disbursement-calculation.bpmn

```diff
@@ -35,7 +37,7 @@ var CSLPPayload = {
   valueCode: "CSLP",
   valueType: "Canada Loan",
   valueAmount: parseFloat(cslptGrant)
-    ? Math.round(parseFloat(cslptGrant) * 100) / 100
+    ? parseFloat(cslptGrant).toFixed(2)
```

Comment:
mm, I believe there was some issue with `.tofixed()` and we had a closed story to move all `.tofixed()` to `Math.round`. may I know, why it was reverted to `.tofixed()`?

Reply:
You are right, took a wrong bpmn deployed. Now fixed, thanks for noting